src/admin/class-wordlift-install-wizard.php

```diff
@@ -0,0 +1,759 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
```

Comment:
The file doc comment must be placed right after the `<?php` tag, see http://www.hashbangcode.com/blog/codesniffer-file-doc-comment-and-class-doc-comment